app/src/main/java/com/ivy/wallet/ui/theme/transaction/TransactionCard.kt

```diff
@@ -110,6 +111,24 @@ fun LazyItemScope.TransactionCard(
 
         }
 
+        if (transaction.description.isNotNullOrBlank()){
+            Spacer(
+                Modifier.height(
+                    if (transaction.title.isNotNullOrBlank()) 4.dp else 8.dp
```

Comment:
Shouldn't it be?
`if (transaction.description.isNotNullOrBlank()) 4.dp else 8.dp`

Reply:
if the description is null or blank the code will not even reach this point at all. the Spacer and the Text are added only if the description is not null or blank and then if there is a title, the space between the title and the description will be 4dp and if not the space will be 8dp which is the space between the description and the header